sdk/core/azure-json/README.md

```diff
@@ -32,9 +32,9 @@ add the direct dependency to your project as follows.
 ### JsonSerializable
 
 `JsonSerializable` is used to define how an object is JSON serialized and deserialized using stream-style serialization
-where the object itself manages the logic for how it's handled with JSON. The interface provides an instance-based
-`toJson` API which handles writing the object to a `JsonWriter` and a static `fromJson` API which implementations must
-override to define how an object is created by reading from a `JsonReader`.
+where the object itself manages the logic for how it's handled. The interface provides an instance-based `toJson` API 
+that handles writing the object to a `JsonWriter` and a static `fromJson` API which implementations must overwrite to 
```

Comment:
We can say "implementations must provide a static `fromJson` method to create an object by reading...". 
For a static method, we don't usually use `overwrite`.

Do we have any checkstyle rule to catch this if an implementation doesn't include the `fromJson` method?

Reply:
I'll file an issue to add a Checkstyle rule to validate implementations of `JsonSerializable` have a `fromJson` method